ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+ • L'implementation d'une technologie propriétaire<br/><br/>
+ • L'utilisation de formats proprietaires<br/><br/>
+ • Le partage de données et la vie privée<br/><br/>
+
+Si votre projet touche à l'un de ces thèmes, ajoutez des étapes supplémentaires
+pour garantir que votre processus est ouvert, inclusif et transparent.
+</td>
+</tr>
+<tr>
+<td><strong>Points clés</strong></td>
+</tr>
+<tr>
+<td>
+ • Confidentialité, vie privée, et contraintes légales<br/><br/>
+ • Controverses potentielles<br/><br/>
+ • Impact sur la culture Red Hat et les futures décisions<br/><br/>
+ • Roles + responsibilités (modèle OPT model: https://github.com/red-hat-people-team/opt-model/) <br/><br/>
+ • Où publier ?
+</td>
+</tr>
+</table>
+
+<br/>
+
+Phase or activity: Planifier, Rechercher <a name="plan-research"></a>
+----------------------
+
+_page 8 sur 15_
+
+<table>
+<tr>
+<td colspan=2 width="66%"><strong>Étapes pour devenir ouverts</strong></td>
+<td><strong>Questions à poser</strong></td>
+</tr>
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Solliciter usagers + collaborateurs</strong><br/>
+ • Rassembler des retours auprès d'usagers internes et de personnes qui vont
+vous aider (questionnaires, entretiens, groupes témoins, etc.)<br/><br/>
+ • Faciliter la participation + les gestion de projet. Demander aux usagers
+quels outils de collaboration ils préfèrent utiliser. Prévoir la consolidation
+et le publication du feedback.<br/><br/>
+ • Rester ouvert à de nouvelles informations et perspectives<Br/><br/>
+ • Envisager le feedback et la commununication pair-à-pair, en plus des canaux
+formels. <br/><br/>
+
+<strong>Définir les attentes dès le départ</strong><Br/>
+ • Etre spécifique à propos des types de feedback que vous recherchez et à
+propos de qui pris les décisions. <Br/><Br/>
+ • Publier le processus de décision et la planification du projet plan avec des
+roles, des dates et des contraintes
+</td>
+<td rowspan=3 width=33%><strong>Expliquer les évidences</strong><Br/>
+ • Publier le périmètre du projet ou de la décision et itérer régulièrement<br/><br/>
+ • Publier les facteurs de décisions et leur importance relative<Br/><Br/>
+ • Publier vos recherches, notamment les compromis difficiles, les pré-requis de
+l'entreprise<br/><br/>
+ • Dans le mesure du possible, publier tous les éléments légaux, les rapports et
+les enjeux de confidentialité<br/><br/>
+
+<strong>Planifier la transition</strong><br/><br/>
+ • Developper et rassembler le feedback sur la communication, la gestion du
+changement et les plans d'adoption<br/><Br/>
+ • Imaginez comment des personnes ennervées pourraient répondre ( via un memo
+ ou un autre canal)
+</td>
+<td>
+ • Comment allons-nous prendre les décisions?<Br/><Br/>
+ • Quel usagers internes, parties prenantes et collaborateurs allons-nous impliquer ?<br/><br/>
+ • Comment allons-nous les solliciter et communiquer avec eux ?<br/><br/>
+ • Quels sont les options open source ?<br/><br/>
```

Comment:
The english version says `open source` which remains `open source` in French (you can find `code ouvert` from time to time but `open source` is the common expression)

`Logiciel libre` is the translation for `Free Software`, which is not used at all in the original version 

There's a debate about the differences between the 2 terms but let's not have this converation here :)